src/main/java/org/sagebionetworks/web/client/presenter/AccessRequirementsForACT.java

```diff
@@ -0,0 +1,209 @@
+package org.sagebionetworks.web.client.presenter;
+
+import static org.sagebionetworks.web.client.ServiceEntryPointUtils.fixServiceEntryPoint;
+
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+import org.sagebionetworks.repo.model.AccessApprovalInfo;
+import org.sagebionetworks.repo.model.AccessRequirement;
+import org.sagebionetworks.repo.model.BatchAccessApprovalInfoRequest;
+import org.sagebionetworks.repo.model.BatchAccessApprovalInfoResponse;
+import org.sagebionetworks.repo.model.RestrictableObjectDescriptor;
+import org.sagebionetworks.repo.model.RestrictableObjectType;
+import org.sagebionetworks.web.client.DataAccessClientAsync;
+import org.sagebionetworks.web.client.PortalGinInjector;
+import org.sagebionetworks.web.client.security.AuthenticationController;
+import org.sagebionetworks.web.client.utils.Callback;
+import org.sagebionetworks.web.client.view.DivView;
+import org.sagebionetworks.web.client.view.PlaceView;
+import org.sagebionetworks.web.client.widget.accessrequirements.AccessRequirementWidget;
+import org.sagebionetworks.web.client.widget.accessrequirements.CreateAccessRequirementButton;
+import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
+import org.sagebionetworks.web.client.widget.table.v2.results.cell.EntityIdCellRenderer;
+import org.sagebionetworks.web.client.widget.team.TeamBadge;
+
+public class AccessRequirementsForACT implements IsWidget {
+
+  private PlaceView view;
```

Comment:
Copy/paste from existing AccessRequirementsPresenter page.  Modified how it is configured - from Place token, to the values parsed from the place token (RestrictableObjectType type, RestrictableObjectDescriptor subject)